docs/topics/generators.md

```diff
@@ -115,7 +115,27 @@ chats = await pipeline.run_many(50)
 
 ## Local Models
 
-We have experimental support for both [`vLLM`](https://docs.vllm.ai/en/latest/) 
+Rigging supports local models via the [`ollama`](https://ollama.ai/) provider:
+
+```py
+import rigging as rg
+
+# default to http://localhost:11434
+ollama = rg.get_generator("ollama/llama3.1")
+```
+
+In the default configuration, the API base address is set to `http://localhost:11434`. It is possible to specify an alternative server by setting the OLLAMA_API_BASE environment variable:
+
+```py
+import os
+import rigging as rg
+
+os.environ['OLLAMA_API_BASE'] = 'http://192.168.0.10:11434'
+
+ollama = rg.get_generator("ollama/llama3.1")
```

Comment:
Can we clarify or add a note about whether ollama_chat is better here? https://docs.litellm.ai/docs/providers/ollama#using-ollama-apichat

Reply:
It's not super clear from their documentation, they only say:

> We recommend using [ollama_chat](https://docs.litellm.ai/docs/providers/ollama#using-ollama-apichat) for better responses.

and 

> In order to send ollama requests to POST /api/chat on your ollama server, set the model prefix to ollama_chat

Should I just add this? I can't tell from the docs in which exact way ollama_chat is better than just ollama.